Gems/Terrain/Assets/Shaders/Terrain/TerrainMacroTextureClipmapGenerationPass.azsl

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include "TerrainMacroTextureClipmapPassSrg.azsli"
+#include "ClipmapComputeHelpers.azsli"
+#include <Atom/RPI/TangentSpace.azsli>
+
+void SampleTexture(uint2 pixelPosition, uint clipmapLevel, out float3 macroColor, out float3 macroNormal)
```

Comment:
There is some issues when converting pixel shader code to compute shader.
But since we will make this into quad tree later, I think it will be fine to leave as it is.